src/components/global/Hero/Hero.jsx

```diff
@@ -0,0 +1,47 @@
+// Material UI Imports
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+import Typography from '@mui/material/Typography';
+//import component specific styles
+import { heroStyles } from './styles';
+
+const Hero = (props) => {
+  let h1Styles = '';
+  let containerStyles = '';
+  switch (props.pageName) {
+    case 'projects':
+      h1Styles = heroStyles.projectsH1;
+      containerStyles = [heroStyles.heroContainerAlt, heroStyles.heroRightAlign];
+      break;
+    case 'volunteer':
+      h1Styles = heroStyles.volunteerH1;
+      containerStyles = [heroStyles.heroContainerAlt, heroStyles.heroCenterAlign];
+      break;
+    default:
+      h1Styles = heroStyles.homeH1;
+      containerStyles = [heroStyles.heroContainer];
+  }
+
+  return (
+    <Box sx={[heroStyles.heroBackground, { backgroundImage: props.heroImage }]}>
+      <Box sx={containerStyles}>
+        <Typography variant="h1" sx={h1Styles}>
+          {props.heroText}
+        </Typography>
+        {props.pageName === 'home' && (
+          <Button
```

Comment:
Since this is a `mailto`, it doesn't need the `target` or `rel` to open in separate window - it opens in whatever email client user has.  I don't BELIEVE they are needed for anything security-wise either, but could be wrong.

Reply:
To my knowledge you are correct. they are not needed for `mailto`. I do want to eventually use a web form for the email to prevent phishing or webscraping spam to Hugh's email. Also a lot of people don't have a default mail app set up on their computer so it's not the nicest solution, just the easiest for now. we can talk more about this at meetup.

Reply:
I removed the target and rel in the last push.